ads/adventive.js

```diff
@@ -139,10 +139,13 @@ function getUrl(context, data, ns) {
  *    representation of the url search query.
  */
 function reduceSearch(ns, placementId, click, referrer) {
-  const isRecipeLoaded = hasOwn(ns.args, 'placementId'),
-      isRecipeStale = !isRecipeLoaded ? true :
-        (Date.now() - ns.args[placementId].requestTime) > (60 * cacheTime),
-      needsRequest = !isRecipeLoaded || isRecipeStale;
+  const isRecipeLoaded = hasOwn(ns.args, 'placementId');
+  let isRecipeStale = true;
+  if (isRecipeLoaded) {
+    const info = ns.args[placementId];
+    isRecipeStale = (Date.now() - info['requestTime']) > (60 * cacheTime);
+  }
+  const needsRequest = !isRecipeLoaded || isRecipeStale;
```

Comment:
This is clearer to follow, but the other methods have similar patterns as the previous code.

Was this change required to get the newer version of CC to correctly compile?

Reply:
Yea, the real change was using brackets instead of dot for dereferencing `requestTime`. CC was complaining that the property did not exist.

Reply:
Might be worth filing an issue to move the rest of this file over to the same approach.